scripts/get_dbdeployer_mysql_json.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Some systems (especially Mac) have an outdated bash in /bin/